demo/src/screens/componentScreens/DateTimePickerScreen.js

```diff
@@ -1,27 +1,42 @@
 import React, {Component} from 'react';
 import {DateTimePicker, View, Text} from 'react-native-ui-lib'; // eslint-disable-line
 
-
 export default class DateTimePickerScreen extends Component {
-  
   render() {
     return (
-      <View flex padding-20>
-        <Text text40 marginB-40>Date Time Picker</Text>
-        
-        <DateTimePicker 
-          title={'Date'} 
-          placeholder={'Select a date'} 
+      <View flex padding-page>
```

Comment:
public repo doesn't have the `page` spacing 
so the example screen will look good in our private environment but in the OSS :( 
```suggestion
      <View flex padding-s5>
```